tests/testconfig.py

```diff
@@ -1,12 +1,12 @@
 from __future__ import annotations
 
-from typing import TYPE_CHECKING, List, Union
+from typing import List, Union
 
-if TYPE_CHECKING:
-    from typing_extensions import Literal
+from typing_extensions import Literal
 
+Oses = Literal["macos", "ubuntu", "windows"]
 # Github actions template config.
-oses = ["ubuntu", "macos"]
+oses: List[Oses] = ["ubuntu", "macos", "windows"]
```

Comment:
Really wish this was `linux`, but i'll let this comment satisfy my desires for now.

```suggestion
oses: List[Oses] = ["macos", "ubuntu", "windows"]
```